integrationTests/__snapshots__/failing.test.js.snap

```diff
@@ -2,8 +2,8 @@
 
 exports[`Works when it has failing tests 1`] = `
 "FAIL integrationTests/__fixtures__/failing/__src__/file1.js
-Company policies require ⚔️ 🏃 in every file
```

Comment:
not sure about this one. Change in jest 22

Reply:
Not sure that I understand why that happened

Reply:
Tracked this down, it's https://github.com/facebook/jest/pull/4504 (which was released in jest 21.2.0)